README.md

```diff
@@ -1,6 +1,6 @@
 ## Usage
 
-Run `npm run start` and visit `localhost:8000` in a browser.  
+Run `npm run start` and visit [`localhost:8000`](localhost:8000) in a browser.  
```

Comment:
maybe add a note about how we use obstacles now. i.e. that obstacles.json is now the single point of truth and that we do not need to redefine them

Reply:
Good idea, will add